src/components/UI/LoadingSpinner.module.css

```diff
@@ -10,8 +10,9 @@
   height: 64px;
   margin: 8px;
   border-radius: 50%;
-  border: 6px solid teal;
-  border-color: teal transparent teal transparent;
+  border: 6px solid var(--text-secondary-color);
```

Comment:
i can't see that LoadingSpinner component in the PR, but I see this in the IDE: ```const LoadingSpinner = () => {
  return <div className={classes.spinner}></div>;
};```
you can use the short notation for react elements that does not have children: 
```const LoadingSpinner = () => {
  return <div className={classes.spinner} />;
};```